Vagrantfile

```diff
@@ -12,9 +12,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Use Ansible as its provisioner     
```

Comment:
Dev environment changes such as the Vagrant and Ansible configuration should be merged back to the dev-env branch too.